about.html

```diff
@@ -0,0 +1,247 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lato-font/3.0.0/css/lato-font.min.css"
+    integrity="sha512-rSWTr6dChYCbhpHaT1hg2tf4re2jUxBWTuZbujxKg96+T87KQJriMzBzW5aqcb8jmzBhhNSx4XYGA6/Y+ok1vQ=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <title>CC common About</title>
+</head>
+
+<body>
+  <div class="after-body">
+
+
+
+    <div id="mobile-menu" class="mobile-menu">
+      <div class="menu-cross">
+        <img onclick="myFunction()" class="cross-btn" src="img/menu-cross.png" alt="close button icon" />
+      </div>
+
+      <ul class="mobile-menu-items">
+        <li>
+          <a onclick="myFunction()" href="index.html">Home</a>
+        </li>
+        <li><a onclick="myFunction()" href="about.html">About</a></li>
+        <li><a onclick="myFunction()" href="#contact-us-email">Contact</a></li>
+      </ul>
+    </div>
+
+
+
+
+
+
+
+
+
+    <div class="social-icon-wrapper display">
+      <div class="social-icons">
+
+        <ul>
+          <li><a href=""><img src="img/twitter.png" alt="twitter icon"></a></li>
+          <li><a href=""><img src="img/victory.png" alt="victory icon"></a></li>
+          <li><a href="">English</a> </li>
+          <li><a href="">My Page</a> </li>
+          <li><a href="">Logout</a> </li>
+        </ul>
+
+      </div>
+    </div>
+
+    <nav class="nav-display-desktop">
+
+      <ul class="nav-grid">
+        <li class="col1"><a href=""><img src="img/logo.png" alt="cc common logo"></a></li>
+        <li class="col2"><a href="index.html">Home</a></li>
+        <li class="col3"><a href="">Program</a></li>
+        <li class="col4"><a href="">Join</a></li>
+        <li class="col5"><a href="">Sponsor</a></li>
+        <li class="col6"><a href="">News</a></li>
+        <li class="col7"><a href="">Campaign</a></li>
+      </ul>
+
+
+    </nav>
+
+
+    <div class="burger-logo display-none">
+      <ul>
+        <li><img onclick="myFunction()" class="burger-icon" src="img/burger-icon.png" alt="menu icon"></li>
+        <li class="logo"> <img src="img/logo.png" alt="logo icon"></li>
+      </ul>
+
+
+    </div>
+
+    <section>
+      <article>
+        <div class="about-wrapper">
+
+          <div class="about-contents">
+            <img class="about-title-text" src="img/about-title.png" alt="title heading text via image">
```

Comment:
- [ ] I noticed that you use an image for this part. It is a bad practice Please make sure that you use a small line of text and for the page title please use a heading tag `hx` for them. Don't forget to use `h1` tag only one time and no more than one time in your both index.html and about.html :+1: 
Example 
```
<hx> Hello Sharing the world <hx>
<h1>CREATIVE COMMONS GLOBAL SUMMIT 2015 <h1>
```